src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -523,7 +533,7 @@ class DocBaseViewer extends BaseViewer {
         let numTicks = ticks;
         let newScale = this.pdfViewer.currentScale;
         do {
-            newScale = (newScale * DEFAULT_SCALE_DELTA).toFixed(3);
+            newScale += DEFAULT_SCALE_DELTA;
```

Comment:
It looks like we may end up losing some rounding here, since we're no longer using `toFixed(3)`. What was the reason behind this change?